app/controllers/cart_items_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CartItemsController < ApplicationController
+  before_action :set_cart_item, only: %i[update destroy] 
+  
+  def index
+    @cart_items = Current.cart.cart_items
+  end
+
+  def create
+    @cart_item = Current.cart.cart_items.new(cart_item_params)
+    @cart_item.quantity = 1
+    if @cart_item.save
+      redirect_back(fallback_location: root_path) 
+    else
+      redirect_back(fallback_location: root_path)
+    end
+  end
+
+  def update
+    if @cart_item.update(quantity: params[:cart_item][:quantity])
+      redirect_back(fallback_location: root_path) 
+    else
+      redirect_back(fallback_location: root_path) 
```

Comment:
Same as create action, why do we need if-else if we are redirecting to the same location?